### PR TITLE
connect to blockchain guide for the init sdk

### PR DESCRIPTION
**Pull request by razacodespython**

### CODE REVIEW

File: guides/connect-to-blockchain/connect-to-blockchain.mdx [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+tags: [sdk, client, server]
+summary: "Providers and Signers are used to connect with the blockchain. Find out how to do this to use our sdk"
+image: "./thumbnail.png"
+date: "2022-02-03"
+updated: ""
+---
+
+## Intro
+
+---
```
Comment (nachoiacovino) [inline guides/connect-to-blockchain/connect-to-blockchain.mdx:16]:
```suggestion

```

why are we using dividers here? can we get rid of all of them?

File: guides/connect-to-blockchain/connect-to-blockchain.mdx [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+
+**RPC** is a simple form of API interaction and stands for **Remote Procedure Call.** In other words we will use this url to make an API call to the chain.
+
+# Using a read only provider with thirdweb
+
+---
+
+The thirdwebs sdk can be used to interact with your modules. To use our sdk to read data from the blockchain we need to create a connection with a `provider`. Then we tell it which blockchain to target via the **RPC** url.
+
+Make sure you have the packages installed. `npm install @3rdweb/sdk` for js and `pip install thirdweb-sdk` for python.
```
Comment (nachoiacovino) [inline guides/connect-to-blockchain/connect-to-blockchain.mdx:46]:
```suggestion
Make sure you have the packages installed. `npm install @3rdweb/sdk` for JavaScript or TypeScript and `pip install thirdweb-sdk` for Python.
```

File: guides/connect-to-blockchain/connect-to-blockchain.mdx [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+const rpcUrl = "https://polygon-rpc.com/";
+const provider = ethers.getDefaultProvider(rpcUrl);
+const sdk = new ThirdwebSDK(provider);
+```
+
+Here’s an example of how to use the provider in an use case where you _just_ want to retrieve data, like getting the number of NFTs you’ve minted so far. \*\*
+
+```jsx
+// Initialize nft module by passing in the module address
+nft_module_address = "0xD9bC403529471515F16DA32eee652c0a6cEcBA78";
```
Comment (nachoiacovino) [inline guides/connect-to-blockchain/connect-to-blockchain.mdx:62]:
```suggestion
const nftModuleAddress = "0xD9bC403529471515F16DA32eee652c0a6cEcBA78";
```

Comment (nachoiacovino) [reply guides/connect-to-blockchain/connect-to-blockchain.mdx:62]: camelCase

File: guides/connect-to-blockchain/connect-to-blockchain.mdx [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+const nft = sdk.getNFTModule(nft_module_address);
```
Comment (nachoiacovino) [inline guides/connect-to-blockchain/connect-to-blockchain.mdx:63]:
```suggestion
const nft = sdk.getNFTModule(nftModuleAddress);
```

File: guides/connect-to-blockchain/connect-to-blockchain.mdx [hunk 5]
```diff
@@ -0,0 +1,148 @@
...
+```python
+# choose your network
+network = "https://rinkeby-light.eth.linkpool.io/"
+# create the object
+sdk = ThirdwebSdk(SdkOptions(), network)
+# use the '.env' file inside Python
+loadenv("./.env")
+PRIVATE_KEY = os.getenv('PRIVATE_KEY')
+# over here we connect our wallet via the private key stored in the '.env' file
+sdk.set_private_key("PRIVATE_KEY")
```
Comment (nachoiacovino) [inline guides/connect-to-blockchain/connect-to-blockchain.mdx:147]:
```suggestion
sdk.set_private_key(PRIVATE_KEY)
```

variable call not string

**Review by nachoiacovino — CHANGES_REQUESTED** [review]
this feels like it should be on learn instead of being a guide?

**Comment by razacodespython** [comment]
I see your point.
for now I would leave it here and then depending on the coming changes with portal, we can move it.

File: guides/connect-to-blockchain/connect-to-blockchain.mdx [hunk 6]
```diff
@@ -0,0 +1,147 @@
...
+draft: false
+tags: [sdk, client, server]
+summary: "Providers and Signers are used to connect with the blockchain. Find out how to do this to use our sdk"
+image: "./thumbnail.png"
+date: "2022-02-03"
+updated: ""
+---
+
+## Intro
+
```
Comment (nachoiacovino) [inline guides/connect-to-blockchain/connect-to-blockchain.mdx:15]:
```suggestion
```

File: guides/connect-to-blockchain/connect-to-blockchain.mdx [hunk 7]
```diff
@@ -0,0 +1,147 @@
...
+
+Connecting and interacting with a blockchain from the frontend works slightly different than you might be used too. In this guide we will dive deeper into what type of connection you need to perform certain interactions with a blockchain.
+
+## Provider
+
+---
```
Comment (nachoiacovino) [inline guides/connect-to-blockchain/connect-to-blockchain.mdx:21]:
```suggestion
---
```

delete all of the dividers under each title

File: guides/connect-to-blockchain/connect-to-blockchain.mdx [hunk 8]
```diff
@@ -0,0 +1,147 @@
...
+);
+```
+
+And here is one in Python 👇
+
+```python
+# choose your network
+network = "https://rinkeby-light.eth.linkpool.io/"
+# create the object
+sdk = ThirdwebSdk(SdkOptions(), network)
```
Comment (nachoiacovino) [inline guides/connect-to-blockchain/connect-to-blockchain.mdx:141]: where is `SdkOptions()* coming from?

shouldn't we also import `ThirdwebSdk`?

**Review by nachoiacovino — CHANGES_REQUESTED** [review]
let's move this under `learn/connect-to-blockchain`

**Pull request merged**